packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Customize the Okta-hosted error pages
+excerpt: Learn how to customize the Okta-hosted error pages to fit your brand.
+layout: Guides
+---
+
+This guide teaches you how to customize error pages that can appear as part of a custom domain Okta-hosted sign-in flow.
```

Comment:
@christinechoy-okta Could you please review the nut facts for this guide? It's my first time having a go at them. Thanks!